### PR TITLE
Support Ruby3

### PR DESCRIPTION
**Pull request by cjcolvar**

### CODE REVIEW

**Comment by joncameron** [comment]
Work on this will be done in the Samvera community sprint.

File: app/models/derivative.rb [hunk 1]
```diff
@@ -22,7 +22,7 @@ class Derivative < ActiveFedora::Base
 
   belongs_to :master_file, class_name: 'MasterFile', predicate: ActiveFedora::RDF::Fcrepo::RelsExt.isDerivationOf
 
-  property :location_url, predicate: ::RDF::Vocab::EBUCore.locator, multiple: false do |index|
+  property :location_url, predicate: ::RDF::Vocab::EBUCore.Locator, multiple: false do |index|
```
Comment (cjcolvar) [inline app/models/derivative.rb:25]: Need to verify that this the same predicate.

File: app/models/derivative.rb [hunk 2]
```diff
@@ -31,12 +31,12 @@ class Derivative < ActiveFedora::Base
   property :duration, predicate: ::RDF::Vocab::EBUCore.duration, multiple: false do |index|
     index.as :stored_sortable
   end
-  property :track_id, predicate: ::RDF::Vocab::EBUCore.identifier, multiple: false
+  property :track_id, predicate: ::RDF::Vocab::EBUCore.Identifier, multiple: false
```
Comment (cjcolvar) [inline app/models/derivative.rb:34]: Need to verify that this the same predicate.

File: app/models/derivative.rb [hunk 3]
```diff
@@ -31,12 +31,12 @@ class Derivative < ActiveFedora::Base
...
   property :hls_track_id, predicate: Avalon::RDFVocab::Derivative.hlsTrackID, multiple: false
   property :managed, predicate: Avalon::RDFVocab::Derivative.isManaged, multiple: false do |index|
     index.as ActiveFedora::Indexing::Descriptor.new(:boolean, :stored, :indexed)
   end
-  property :derivativeFile, predicate: ::RDF::Vocab::EBUCore.filename, multiple: false do |index|
+  property :derivativeFile, predicate: ::RDF::Vocab::EBUCore.resourceFilename, multiple: false do |index|
```
Comment (cjcolvar) [inline app/models/derivative.rb:39]: Need to verify that this the same predicate.

File: app/models/master_file.rb [hunk 4]
```diff
@@ -57,7 +57,7 @@ class MasterFile < ActiveFedora::Base
   property :title, predicate: ::RDF::Vocab::EBUCore.title, multiple: false do |index|
     index.as :stored_searchable
   end
-  property :file_location, predicate: ::RDF::Vocab::EBUCore.locator, multiple: false do |index|
+  property :file_location, predicate: ::RDF::Vocab::EBUCore.Locator, multiple: false do |index|
```
Comment (cjcolvar) [inline app/models/master_file.rb:60]: Need to verify that this the same predicate.

File: app/models/master_file.rb [hunk 5]
```diff
@@ -88,7 +88,7 @@ class MasterFile < ActiveFedora::Base
   property :physical_description, predicate: ::RDF::Vocab::EBUCore.hasFormat, multiple: false do |index|
     index.as :stored_sortable
   end
-  property :masterFile, predicate: ::RDF::Vocab::EBUCore.filename, multiple: false
+  property :masterFile, predicate: ::RDF::Vocab::EBUCore.resourceFilename, multiple: false
```
Comment (cjcolvar) [inline app/models/master_file.rb:91]: Need to verify that this the same predicate.

File: .circleci/config.yml [hunk 6]
```diff
@@ -5,7 +5,7 @@ jobs:
   build:
     docker:
       # Primary container image where all steps run.
-      - image: avalonmediasystem/avalon:7.3.0-dev
+      - image: avalonmediasystem/avalon:7.5.0-dev-ruby3
```
Comment (cjcolvar) [inline .circleci/config.yml:8]: Once this PR is merged a new image will be built and pushed to docker hub by bamboo so I'll change this to that image in a subsequent PR.

**Comment by cjcolvar** [comment]
Closing since this is included in #5058

**Pull request closed without merging**